packages/framework-provider-azure/src/index.ts

```diff
@@ -41,6 +44,25 @@ if (typeof process.env[environmentVarNames.cosmosDbConnectionString] === 'undefi
   cosmosClient = new CosmosClient(process.env[environmentVarNames.cosmosDbConnectionString] as string)
 }
 
+let producer: EventHubProducerClient
+const eventHubConnectionString = process.env[environmentVarNames.eventHubConnectionString]
+const eventHubName = process.env[environmentVarNames.eventHubName]
+if (
+  typeof eventHubConnectionString === 'undefined' ||
+  typeof eventHubName === 'undefined' ||
+  eventHubConnectionString === '' ||
+  eventHubName === ''
+) {
+  producer = {} as any
+} else {
+  producer = new EventHubProducerClient(eventHubConnectionString, eventHubName, {
+    retryOptions: {
+      maxRetries: 5,
+      mode: RetryMode.Exponential,
```

Comment:
Not sure if it will be possible but, Could we not include this in the configuration?

Reply:
You don't want this in the configuration?

Reply:
@gonzalogarciajaubert HAHAHA Now that I read the question, I though exactly the same as you, which is exactly the opposite of what I wanted to ask XD. 

What I meant is to _include_ this in the configuration. Many projects would probably want to configure this, and it is generic enough (not strictly associated to Azure) to include it in the config